src/aap_eda/api/views/root.py

```diff
@@ -70,3 +71,12 @@ def list_urls(urls):
         urls = urls.eda_v1_urls
 
     return list_urls(urls)
+
+
+class ApiV1ConfigView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        from aap_eda import __version__
+
```

Comment:
@Dostonbek1 We might need to send back more information like deployment type etc since that helps the UI team block out UI elements specific to deployment type.
https://github.com/ansible/eda-server/blob/55e8897c3be3e7f537fe0b1c9ee219303fbc6d07/src/aap_eda/settings/default.py#L340 maybe we can create a Dataclass with the required fields and send that out as json.

Reply:
yup, agreed. I was mainly trying to capture the minimum requirement asked in this ticket but I think adding extra data shouldn't be too hard and we can include them in this PR. Here is the extra data we can include:
```
- version
- deployment type (podman or kubernetes)
- controller url
```
can you think of anything else we should add?

Reply:
@Dostonbek1 The current config end point in AAP has the following data
/api/v2/config/
```
{
    "time_zone": "UTC",
    "license_info": {
        "license_type": "open",
        "valid_key": true,
        "subscription_name": "OPEN",
        "product_name": "AWX"
    },
    "version": "24.1.1.dev19+gb022b50966.d20240404",
    "eula": "",
    "analytics_status": "off",
....

```

Reply:
@Dostonbek1 Can we check with the UI team which end point they would like to call and also where should the version string be coming from. The installer team might be using a different value. This should work for both upstream and downstream.

Reply:
@Dostonbek1 Controller URL is being deprecated in favor of RH AAP Credential type it won't be there going forward